config/kibana.yml

```diff
@@ -11,6 +11,9 @@
 # The maximum payload size in bytes for incoming server requests.
 # server.maxPayloadBytes: 1048576
 
+# The Kibana server's name.  This is used for display purposes.
```

Comment:
Given that we don't actually show this anywhere, this is a weird description. Can you add the display of this to the status and about screens of kibana?